### PR TITLE
[LCS-282] - Customer email

### PR DESCRIPTION
**Pull request by lennym**
Sends an email to the applicant after a successful submission.

### CODE REVIEW

File: apps/right-to-rent-check/emails/customer.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  {{/tenants}}
+</ul>
+
+<p>We will carry out this check and email you within 2 working days with a unique reference number. Our decision will be based on the details you've submitted (shown below). Please keep this email as proof of your request.</p>
+
+<p>If any of these details are wrong, or you have received this email in error, call us on 0300 069 9799. Please do not call us to check up on your application as we will not be able to process it any faster.</p>
+
+<p>View the <a href="https://www.gov.uk/check-tenant-right-to-rent-documents">right to rent</a> guidance.</p>
+
+<h2>Your confirmation details</h2>
```
Comment (JoeChapman) [inline apps/right-to-rent-check/emails/customer.html]: Can we stick these words in a translations file?

File: apps/right-to-rent-check/emails/customer.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  <p>If you have any further queries about right to rent, contact the landlord’s helpline. Please note, we can’t discuss individual applications.</p>
+  <h4>Landlord's helpline</h4>
+  <p>
+    0300 069 9799<br/>
+    Monday to Thursday 9am to 4:45pm<br/>
+    Friday 9am to 4:30pm
+  </p>
+</div>
+
+<p>If you have any concerns regarding the validity of this or any emails received from Home Office go to our <a href="">Online Security pages</a></p>
```
Comment (JoeChapman) [inline apps/right-to-rent-check/emails/customer.html]: And all these words, shouldn't they be translated

File: apps/right-to-rent-check/behaviours/customer-email.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+const getDataRows = model => {
+  const isAgent = model.representative === 'agent';
+  return [
+    {
+      table: [
+        { label: 'Date and time of submission', value: moment().format('DD-MM-YYYY, hh:mma') },
+        { label: 'Rental property address', value: model['rental-property-address'] },
+        { label: 'Current property address', value: model['current-property-address'] }
+      ]
```
Comment (JoeChapman) [inline apps/right-to-rent-check/behaviours/customer-email.js]: @lennym shouldn't the label's be translations?

**Comment by lennym** [comment]
I've updated this to move any content that was in code out into translation files.

**Pull request merged**